src/bin/parser.rs

```diff
@@ -5,5 +5,9 @@ extern crate uosql;
 use uosql::parse;
 
 fn main() {
-    println!("{:?}", parse::parse("hi"));
+	let l = parse::lex::Lexer::from_query("Create table personen (name_v var(256), name_n var(256), alter char(16));");
+	let m = parse::lex::Lexer::from_query("\'10.08.1991\" and this does not belong to it");
```

Comment:
Just as a note: Rust has raw string literals in which you don't need to escape chars like " and '. Just write it like that:
```
let s = r#""10.08.1991" and this does not belong to it"#;
```
In raw string literals you can choose your delimiter ("#" in this case) that is needed to end the string. The delimiter obviously wasn't  appear in the string contents. So if you want to write the string
```
#yolo #"hashtag
```
In rust, the delimiter "#" does not work. We could, for example, just use "~" as delimiter for this string:
```
let s = r~"#yolo #"hashtag"~;
```